sdcm/results_analyze.py

```diff
@@ -116,12 +116,14 @@ def _test_stats(self, test_doc):
         return stats_average
 
     def _test_version(self, test_doc):
-        if 'versions' not in test_doc['_source'] or 'scylla-server' not in test_doc['_source']['versions']:
-            logger.error('Scylla version is not found for test %s', test_doc['_id'])
-            return None, None
+        if test_doc['_source'].get('versions'):
+            for v in ('scylla-server', 'scylla-enterprise-server'):
+                k = test_doc['_source']['versions'].get(v)
+                if k:
+                    return k
 
-        return (test_doc['_source']['versions']['scylla-server']['version'],
-                test_doc['_source']['versions']['scylla-server']['date'])
+        logger.error('Scylla version is not found for test %s', test_doc['_id'])
+        return None
```

Comment:
Python returns None by default

Reply:
If method returns value in a normal case, I prefer return explicitly in a bad case.